src/effects/browse.js

```diff
@@ -12,3 +12,13 @@ export function fetchSchools() {
       querySnapshot => querySnapshot.docs.map(doc => doc.data()),
     );
 }
+
+export function fetchSubjects(termId, subjectId) {
+  return currentTermDoc
+    .collection('subjects')
+    .where('subjectId', '==', subjectId)
```

Comment:
this should be `.where(`id`, '==', subjectId)` since for the subjects objects, the identifier will just be `id`. See https://github.com/Joonpark13/sans-serif#subject